docs/decisions/0010-removal-of-edx-assets.rst

```diff
@@ -0,0 +1,76 @@
+5. Removal of edX.org assets
+----------------------------
+
+
+Status
+------
+
+Accepted
+
+
+Context
+-------
+
+Paragon currently contains:
+  - SCSS framework based on Bootstrap
+  - edX SCSS theme
+  - React components
+  - Documentation site
+
+Merges to master trigger work to:
+  - Publish @edx/paragon to npm.
+  - Build and deploy Paragon documentation with the edX.org theme to https://edx.github.io/paragon
+
+There are pros and cons to this project's current structure. First the pros:
+  - The edX theme is represented in the doc site with little technical complexity. This is good for edX employees to understand what our visual styles are.
+  - edX theme changes are easily seen during development of components.
+
+Cons:
+  - edX theme changes are coupled with Paragon component changes. This can block teams from upgrading the edX theme in their projects.
+  - The default theme is not represented in documentation.
+  - This project is open source but not for general use by default.
+
+
+Decision
+--------
+
+We will create a new project for edX.org branding. It will include the edX theme and other edX brand assets. Paragon will no longer have any specific tie to edX,org visuals. The new structure
+
+Paragon:
+  - SCSS framework based on Bootstrap
+  - React components
+  - Documentation site
+
+Merges to Paragon master trigger work to:
+  - Publish @edx/paragon to npm.
+  - Build and deploy Paragon documentation with no theme to https://edx.github.io/paragon
+  - Build and deploy a version of the Paragon documentation with the edX.org theme at https://paragon.edx.org
+
+edX Brand:
+  - edX theme
+  - edX brand assets (e.g. logo.svg)
+
+Merges to edX Brand trigger work to:
+  - Build and deploy a version of the Paragon documentation with the edX.org theme at https://paragon.edx.org
+
+This structure has pros and cons as well:
+
+Pros:
+  - The edX theme is decoupled from Paragon. Adopting theme updates is easier for teams.
+  - The default and edX themes are both documented.
+  - There is a new common location for brand assets.
+  - Paragon is an independent open source library
+
+Cons:
+  - Two documentation sites one for edX and one for Paragon makes for some additional CI complexity.
+  - edX.org theme changes may be more difficult to see during development. We will need to determine an efficient way to do this.
```

Comment:
This is my biggest worry... that this will make developing components with final theme in mind more complicated...

Reply:
Fair concern. I think this can be mitigated (https://github.com/edx/frontend-build/pull/103) and is worth the trade off. 

Also, right now it's difficult for engineers to know what the component looks like without a theme. So we have the reverse challenge (though not hard to test out).

Reply:
Updated with more detail in the consequences section